spec/controllers/scenarios_controller_spec.rb

```diff
@@ -140,6 +140,17 @@ def valid_attributes(options = {})
     end
   end
 
+  describe 'PUT enable_or_disable_all_agents' do
+    it 'updates disabled on all agents in a scenario for the current user' do
+      #@id = scenarios(:bob_weather).id
+      #@scenario = { :scenario => { :disabled => 'true' } }
+      #put :enable_or_disable_all_agents, id: @id, scenario: @scenario
+      put :enable_or_disable_all_agents, :id => scenarios(:bob_weather), :scenario => { :scenario => { :disabled => true } }
+      expect(agents(:bob_rain_notifier_agent).disabled).to eq(true)
+      expect(response).to redirect_to(scenario_path(scenarios(:bob_weather)))
+    end
+  end
+
```

Comment:
I am not sure why this test isn't passing. The `params` are in `hash` and it should be getting `put` correctly. I also tried `post`. Locally it is fine but the test just isn't passing.